Formula/apophenia.rb

```diff
@@ -0,0 +1,20 @@
+class Apophenia < Formula
+  desc "C library for statistical and scientific computing"
+  homepage "http://apophenia.info/"
+  url "https://github.com/b-k/apophenia/archive/v1.0.tar.gz"
+  sha256 "c753047a9230f9d9e105541f671c4961dc7998f4402972424e591404f33b82ca"
+
+  depends_on "gsl"
+  depends_on "sqlite"
+
+  def install
+    system "./configure", "--with-mysql=no", "--enable-extended-tests", "--prefix=#{prefix}"
+    system "make", "-j"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"foo").write("1")
+    system "#{bin}/apop_text_to_db", "foo", "bar", "baz"
+  end
```

Comment:
@apjanke,  This is basically a C library, so the test I built here is similar to the one for tinyxml2, as referred to in the Formula Cookbook.   There are three command line utilities provided with the library and I test that one of them can/has built correctly and accepts a sensible input.